src/server/scripts/Commands/cs_reload.cpp

```diff
@@ -414,7 +414,12 @@ class reload_commandscript : public CommandScript
         LOG_INFO("server.loading", "Reloading Motd...");
         sMotdMgr->LoadMotd();
         handler->SendGlobalGMSysMessage("DB table `motd` reloaded.");
-        handler->SendGlobalSysMessage(sMotdMgr->GetMotd());
+        auto locale = DEFAULT_LOCALE;
```

Comment:
I dont think we need to use auto here. We know it will always be LocaleConstant anyways